### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by VMBindraban** [serserm/react-native-turbo-serialport#6]
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:processSomeAppDebugResources'.
> Could not resolve all dependencies for configuration ':app:someappDebugRuntimeClasspath'.
   > Could not find com.github.felHR85:UsbSerial:6.1.0.
     Searched in the following locations:
       - https://oss.sonatype.org/content/repositories/snapshots/com/github/felHR85/UsbSerial/6.1.0/UsbSerial-6.1.0.pom
       - https://repo.maven.apache.org/maven2/com/github/felHR85/UsbSerial/6.1.0/UsbSerial-6.1.0.pom
       - https://dl.google.com/dl/android/maven2/com/github/felHR85/UsbSerial/6.1.0/UsbSerial-6.1.0.pom
       - https://www.jitpack.io/com/github/felHR85/UsbSerial/6.1.0/UsbSerial-6.1.0.pom
     Required by:
         project :app > project :serserm_react-native-turbo-serialport

```

Using V2.2.1.

**Comment by VMBindraban** [comment]
Changing the `build.gradle` to `implementation "com.github.felHR85:UsbSerial:6.0.6"` works.

**Comment by Shakahs** [comment]
https://github.com/felHR85/UsbSerial/issues/375

**Comment by scalhoun** [comment]
Would dropping to `com.github.felHR85:UsbSerial:6.0.6` cause any issues with the event listeners?  

I can connect and write data but none of the event listeners from the examples are triggering properly.

**Comment by VMBindraban** [comment]
> Would dropping to `com.github.felHR85:UsbSerial:6.0.6` cause any issues with the event listeners?
> 
> I can connect and write data but none of the event listeners from the examples are triggering properly.

I noticed that some events are not triggered, i am not sure if it's the device fault or not.

For example i only get a respond when the tv source is changed when using serial.
I do get a response when the tv turns on. (remote or serial)

**Comment by SerSerch** [comment]
Hi! fixed 2.2.2
